lib/netsuite/records/expense_category.rb

```diff
@@ -0,0 +1,20 @@
+module NetSuite
+  module Records
+    class ExpenseCategory
+      include Support::Fields
+      include Support::Actions
+      include Namespaces::ListAcct
+
+      actions :search
```

Comment:
Can you add the reference link to be consistent with other record files?
https://system.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2014_1/script/record/expensecategory.html